src/network/networkd-network-gperf.gperf

```diff
@@ -35,6 +35,7 @@ Link.MTUBytes,                          config_parse_iec_size,
 Link.ARP,                               config_parse_tristate,                          0,                             offsetof(Network, arp)
 Link.Unmanaged,                         config_parse_bool,                              0,                             offsetof(Network, unmanaged)
 Link.RequiredForOnline,                 config_parse_bool,                              0,                             offsetof(Network, required_for_online)
+Link.Shutdown,                          config_parse_bool,                              0,                             offsetof(Network, shutdown)
```

Comment:
Can we find a different name  for this? maybe KeepDown=, CarrierOff=, Offline= or so? "Shutdown" in the systemd world has been used primarily for stuff related to system shutdown, hence I'd avoid it in this new context.

Reply:
It's common Cisco parlance, so that's what I used. I'm not attached to any particular name. Offline= or KeepDown=  maybe?

Reply:
Offline= sounds good to me.

Reply:
KeepDown= too